core/domain/learner_playlist_services_test.py

```diff
@@ -100,162 +101,128 @@ def _get_all_learner_playlist_collection_ids(self, user_id):
             learner_playlist_model.collection_ids if
             learner_playlist_model else [])
 
-    def test_mark_exploration_to_be_played_later(self):
+    def test_adding_subscribed_exploration_to_playlist_does_not_add_again(self):
+        # Subscribe to exploration.
+        subscription_services.subscribe_to_exploration(
+            self.user_id, 'SUBSCRIBED_ID')
```

Comment:
Here and below, I am a bit confused. What is 'SUBSCRIBED_ID'? It does not seem to actually correspond to a valid thing.

Shouldn't you be:
- using an actual exp ID here
- checking that bad IDs result in errors being raised?

Reply:
The previous developer on this file had used this notation. Also, the subscribe_to_exploration function in subscription_services appends whatever ID is inputted to a list. So it wouldn't matter whether the ID is actually valid. 
SUBSCRIBED_ID simply represents a placeholder for the name of any ID.